src/Files.App/Utils/Storage/Operations/FilesystemHelpers.cs

```diff
@@ -256,11 +256,9 @@ public async Task<ReturnResult> RestoreItemsFromTrashAsync(IEnumerable<IStorageI
 				else if (operation.HasFlag(DataPackageOperation.Link))
 				{
 					// Open with piggybacks off of the link operation, since there isn't one for it
-					if (isTargetExecutable || isTargetPythonFile)
+					if (isTargetExecutable || isTargetScriptFile)
 					{
 						var items = await GetDraggedStorageItems(packageView);
-						if (isTargetPythonFile && !SoftwareHelpers.IsPythonInstalled())
```

Comment:
What happens if the target isn't installed?

Reply:
https://github.com/files-community/Files/pull/14895#issuecomment-1975617972

Reply:
Looking back, I think the comment was unclear. There is no requirement for AutoHotkey to be in the PATH. The program will still function as intended. This picture is if AutoHotkey is neither installed nor in the PATH.

Reply:
I meant, what happens if Python isn't installed?

Reply:
From my testing, it treats the file as if you would normally try to open a .py without python installed. If no windows default application for opening the file type is defined, it'll prompt for one like the picture suggests. I'm not sure if I understand or have answered your question correctly, I apologize.